packages/zowe-explorer/package.json

```diff
@@ -9,6 +9,7 @@
   "repository": {
     "url": "https://github.com/zowe/vscode-extension-for-zowe"
   },
+  "enabledApiProposals": [],
```

Comment:
Do we want to add this line, or is it left over from trying out VS Code's API proposal for quick pick separators? 😋